src/menu/charprofile.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * This software is licensed under the terms of the MIT License.
+ * See COPYING for further information.
+ * ---
+ * Copyright (c) 2011-2019, Lukas Weber <laochailan@web.de>.
+ * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
+ */
+
+#pragma once
+#include "taisei.h"
+
+#include "menu.h"
+#include "portrait.h"
```

Comment:
`#include "portrait.h"` should go into the `.c` file, because nothing in the header uses it.